esrally/metrics.py

```diff
@@ -1396,8 +1396,8 @@ def __init__(self, es_store, es_results_store, file_store):
         self.es_results_store = es_results_store
         self.file_store = file_store
 
-    def find_by_timestamp(self, timestamp):
-        return self.es_store.find_by_timestamp(timestamp)
+    def find_by_trial_id(self, trial_id):
```

Comment:
Do we need this commit? My understanding is that when it comes to storing things in the `rally-races-*` index, we store both trial-id and trial-timestamp. Additionally `rally list races` still shows the timestamp (as well as `night-rally-admin list races`) so I thought the intent of https://github.com/elastic/rally/issues/697 is just about the installation directory:

>  Switch from installation directories based on the race timestamp to installation directories based on unique ids (~/.rally/benchmarks/nodes/$ID)

Reply:
paths.race_root was modified to use trial_id instead of timestamp, which broke the previous implementation of find_by_timestamp for FileRaceStore.  Strictly speaking, there's no reason the EsRaceStore couldn't keep its find_by_timestamp functionality (as it doesn't use this function), but I thought it was cleaner to make it consistent with FileRaceStore and the abstract RaceStore.